### ISSUE TITLE
Element with role has required states and properties - Failed example 1 [4e8ab6]

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1373]
More ARIA versioning issues. 

```
<div role="combobox" aria-expanded="true"></div>
```
https://act-rules.github.io/rules/4e8ab6#failed-example-1

**ARIA 1.0 combobox pattern** - `aria-expanded` always required , `aria-owns` not listed as required - should be used when aria-autocomplete=list (but should rather than must)
https://www.w3.org/TR/wai-aria-1.0/roles#combobox
https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-6

**ARIA 1.1 combobox pattern** - `aria-expanded` always required, `aria-controls` only set to set to popup element
when expanded
https://www.w3.org/TR/wai-aria-1.1/#combobox
https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-6

**ARIA 1.2 combobox pattern** - `aria-expanded` and `aria-controls` always required  with `aria-controls` always set to popup element
https://www.w3.org/TR/wai-aria-1.2/#combobox

The problem is that the ARIA 1.1 spec supports both the 1.0 and 1.1 combobox:

> The ARIA 1.0 specification describes a combobox pattern where a text input element has the combobox role and owns a listbox element. User agents, assistive technologies, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional.
https://www.w3.org/TR/wai-aria-1.1/#combobox

**tl;dr;** until ARIA 1.2 moves to CR the only required attribute for `role=combbox` is `aria-expanded` because ARIA 1.1 supports both the 1.1 and 1.0 combobox patterns, so failed example 1 is a pass

**Comment by WilcoFiers** [comment]
I tend to agree. I think we should leave this rule for what it is until ARIA 1.2 makes it to rec. At the moment, the only required prop that doesn't have a default, or some caveat that allows an out in ARIA 1.1 is `aria-controls` on `scrollbar`. ARIA 1.2 is much stricter, and appropriately so I think, but I think we're going to have to wait for that to be published before we put more work into this rule. That also means we're not taking the rule to the TF any time soon.

**Comment by WilcoFiers** [comment]
Decision from the CG: leave this rule as is, knowing there is an issue with it, until ARIA 1.2 is out. We're not passing this rule onto the TF yet.

**Comment by dd8** [comment]
Some useful background reading on the design process that led to the combobox 1.2 pattern, and the problems with the 1.1 pattern:

https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues

**Comment by Jym77** [comment]
This was closed by #1884
